src/main/java/seedu/address/model/person/patient/DateOfBirth.java

```diff
@@ -1,7 +1,88 @@
 package seedu.address.model.person.patient;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.stream.Stream;
+
 /**
  * Represents a patient's date of birth in the patient record
  */
 public class DateOfBirth {
+    private static final String MESSAGE_CONSTRAINTS =
+            "Date of birth should only contain numeric characters and spaces, "
```

Comment:
small type here, I think you mean 01/01/1900

Reply:
Fixed.